packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -413,13 +413,15 @@ contract Deploy is Deployer {
 
         setupCustomGasToken();
 
+        address storageSetter = deployStorageSetter();
+
         // Selectively initialize either the original OptimismPortal or the new OptimismPortal2. Since this will upgrade
         // the proxy, we cannot initialize both.
         if (cfg.useFaultProofs()) {
             console.log("Fault proofs enabled. Initializing the OptimismPortal proxy with the OptimismPortal2.");
             initializeOptimismPortal2();
```

Comment:
Should we also use this pattern for the Portal2 already? 
I think it would be good to include this before the code-freeze, since then we also cover the upgrade script for fault-proofs (see https://github.com/celo-org/optimism/issues/239)

Reply:
Lets make it part of the task #239